src/app/app.module.ts

```diff
@@ -104,4 +102,17 @@ export function markedOptions(): MarkedOptions {
 
   ]
 })
-export class AppModule { }
+export class AppModule {
+  // work around for ivy tree shaking these out in the production build
+  // https://github.com/angular/angular/issues/35314#issuecomment-584821399
+  static  entryComponents = [
+    ArrayFilterComponent,
+    BriefExporterComponent,
+    CustomColumnFilterComponent,
+    CustomExporterComponent,
+    SimpleFilterComponent,
+    PropertyOptionsComponent,
+    SelectionExporterComponent
+
+  ]
+}
```

Comment:
This is the work around to prevent the example components to be tree shaken out of the production build code.  

Essentially the old way of listing them as part of the @NgModule declaration works differently in ivy.